versions.toml

```diff
@@ -1,4 +1,4 @@
 # Update these values when a new release is created.
 # `stacks-common/build.rs` will automatically update `versions.rs` with these values.
-stacks_node_version = "3.1.0.0.7"
-stacks_signer_version = "3.1.0.0.7.0"
+stacks_node_version = "3.1.0.0.8-rc1"
+stacks_signer_version = "3.1.0.0.8.0-rc1"
```

Comment:
Hm, since you previously had built this off of .8, you have this artifact (also the changelog). Not sure if it's worth removing it from this PR if we know it'll get included in .8

Reply:
we can address this separately, good eye!